tests/python_client/testcases/test_collection.py

```diff
@@ -2904,6 +2904,315 @@ def test_load_collection_without_creating_index(self):
                           check_items={"err_code": 1,
                                        "err_msg": "index not exist"})
 
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_delete_load_collection_release_partition(self):
```

Comment:
I think search for all these scenario are needed too.

Reply:
I have added search for all these scenarios.